aeneid-testnet/hook-modules.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "LockLicenseHook",
+    "address": "0x54C52990dA304643E7412a3e13d8E8923cD5bfF2",
+    "blockExplorerLink": "https://aeneid.storyscan.io/address/0x54C52990dA304643E7412a3e13d8E8923cD5bfF2?tab=contract",
+    "whitelisted": true
+  },
+  {
+    "name": "TotalLicenseTokenLimitHook",
+    "address": "0xaBAD364Bfa41230272b08f171E0Ca939bD600478",
+    "blockExplorerLink": "https://aeneid.storyscan.io/address/0xaBAD364Bfa41230272b08f171E0Ca939bD600478?tab=contract",
+    "whitelisted": true
+  },
+  {
+    "name": "IPUSDPriceHook",
```

Comment:
- If Pyth malfunctions, shut downs, etc, what are the steps for IP IDs using the hook? We would have to pause LicensingModule? How would delisting this hook and whitelisting a new one affect IP IDs that were using it?
- Pyth price structure has confidence values for sanity check. We should have a threshold and check that we don't use prices with insufficient confidence.
- Pyth prices must be normalized to 18 decimals